snmp/unpriv/osupdates/osupdates-unpriv-generate.sh

```diff
@@ -29,7 +29,7 @@ BIN_PKG='/usr/sbin/pkg'
 CMD_PKG=' audit -q -F'
 BIN_APK='/sbin/apk'
 CMD_APK=' version'
-SNMP_PERSISTENT_DIR="$(net-snmp-config --persistent-directory)"
+SNMP_PERSISTENT_DIR="/var/lib/net-snmp"
```

Comment:
Oh, perhaps not.  but why?

Reply:
So Fedora doesn't have it, so the rest of the OS must accommodate it?

Why is Fedora the chosen correct install path?